packages/turbo/ios/RNSessionConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RNSessionConfiguration.swift
+//  RNTurbo
+//
+//  Created by Patryk Klatka on 22/04/2024.
+//
+
+import Foundation
+
+class RNSessionConfiguration {
+  var sessionDelegate: RNSession? = nil {
```

Comment:
`sessionDelegate`? It's not really a session _delegate_ here, right? It's the actual `session`